### DISCUSSION TITLE
Overwriting screen sizes retains old break points with screen.

### DISCUSSION THREAD

**Discussion by CHAscend** [tailwindlabs/tailwindcss#13957] [Help] [answered]
I'm using "version": "3.4.4" within a laravel, vue and vite project. 

I'm having an issue when overwriting the existing screen sizes in tailwind config that the original break points are still being applied on some definitions.

In tailwind.config I have
 
`export default {
  theme: {
    screens: {
      sm: '600px',
      md: '905px',
      lg: '1240px',
      xl: '1600px',
    },`

When I apply `xl:ml-4` to a div it generates breakpoints on the old 1200px.

![Screenshot 2024-07-05 at 13 46 23](https://github.com/tailwindlabs/tailwindcss/assets/143753259/d9db727b-046e-4629-83d0-593e63ad1b9f)

What makes this odd is that on the same page, the xl: applied to `xl:h-32 xl:w-32 xl:ml-20` creates breakpoints at the expected 1600px.

![Screenshot 2024-07-05 at 13 48 51](https://github.com/tailwindlabs/tailwindcss/assets/143753259/8a7f349f-beae-491c-8053-8c61210f4965)

For the xl:ml4 it seems to be creating a screen and min width break point rather than just min width.

I was just looking for some advice if this is expected behaviour or if Im doing something wrong here?

Thanks

**Comment by wongjn** [comment, marked as answer]
The default for `xl` is `1280px`, not `1200px`. Furthermore, the erroneous media queries have `screen and` whereas the correct media queries do not. It seems like these rules may be coming from some other CSS in your project that you have not mentioned.

**Reply by CHAscend** [reply]
@wongjn thanks for the reply.  

I've an idea what this could be, I'll investigate further and report back what trouble I've caused myself.

**Reply by CHAscend** [reply]
Yeah, these were being pulled in from :  `@use 'primeflex/primeflex.min.css';`
